spec/models/solidus_friendly_promotions/promotion_rule_spec.rb

```diff
@@ -3,11 +3,16 @@
 require "spec_helper"
 
 RSpec.describe SolidusFriendlyPromotions::PromotionRule do
-  class BadTestRule < SolidusFriendlyPromotions::PromotionRule; end
-
-  class TestRule < SolidusFriendlyPromotions::PromotionRule
-    def eligible?(_promotable, _options = {})
-      true
+  let(:bad_test_rule_class) { Class.new(SolidusFriendlyPromotions::PromotionRule) }
+  let(:test_rule_class) do
+    Class.new(SolidusFriendlyPromotions::PromotionRule) do
+      def self.model_name
+        ActiveModel::Name.new(self, nil, "test_rule")
+      end
+
+      def eligible?(_promotable, _options = {})
+        true
+      end
```

Comment:
The tests still pass after removing this method. Do you want to add a positive test for test_rule_class or maybe an integration type spec that uses this? Or just remove it (but that might look stranger ...)?

Reply:
Added a spec that tests the good test rule being a good one.